charms/kserve-controller/config.yaml

```diff
@@ -1,6 +1,14 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 options:
+  deployment-mode:
+    default: "RawDeployment"
+    description: Deployment mode for kserve. It can only be RawDeployment or Serverless, and the latter requires knative-serving to be present in the model, and the local-gateway relation to be established.
+    type: str
+  domain-name:
```

Comment:
can we check in the code that it is the same as in knative ?

Reply:
It's a bit hard because knative-serving is not sharing that data, for that I would need to share it somehow and that would require to either modify the gateway info library or create a new one just for knative.

Reply:
let's treat it as the enhancement which is out of this scope. Can you open an issue to track it?

Reply:
Tracked in #98